spec/index.md

```diff
@@ -335,163 +325,131 @@ Each dimension is represented by a key in the `dimensions` object. The object ha
 
-## Special Dimensions
+## Classification
 
-### Classifications
-
-It is common for fiscal data to be classified in various ways. A classification is a labelling of a given item with a reference to standardized codesheet.
-
-Classifications will be represented in the model as a dimension. Each classification dimension `MUST` have a `code` attribute whose value will correspond to the classification code in the official codesheet. Sometimes classifications can change and we recommend utilizing a `version` attribute if there is a need to indicate the version of a classification.
-
-Whenever we have a code attribute in a classification dimension, the licit values for that attribute consist of the numerical codes from the appropriate codesheet, with hierarchical levels separated by periods. `1.1.4.1.3` is a licit value for a dimension named `gfsm`, for example, corresponding to the code for "Turnover and other general taxes on goods and services".
-
-Classifications are of different types. The type of the classification `MAY` be indicated using the `classificationType` attribute on the dimension. Values are:
-
-* `functional`
-* `administrative`
-* `economic`
-
-#### Hierarchical Classifications
-
-It is common for classifications to be hierarchical and have different levels. For example, a functional classification might include a top-level of "Healthcare" and a sub-level under "Healthcare" of "Hospitals".
-
-This hierarchical structure of the classification can be recorded using the keyword `parent`.  The `parent` keyword is used within an attribute definition to reference another attribute in the same dimension that is the parent of the first attribute. Here is an example:
+```javascript
+"spending-classification": {
+  "dimensionType": "classification",
+
+  // RECOMMENDED: The basis on which transactions are being classified, one of these values:
+  //
+  // * "administrative": an organisational structure, such as 
+  //   Portfolio > Department > Branch
+  // * "functional": the purpose of the spending, such as 
+  //   Health > Hospital services > Nursing
+  // * "economic": focused on the nature of the accounting, such 
+  //   as Compensation > Wages and salaries > Wages and salaries in cash
+  "classificationType": "administrative"
 
-```
-"your-classification": {
   "attributes": {
+    // REQUIRED: a "code" column containing unique identifiers from an official codesheet. If classifications are subject to change, a `version` attribute `SHOULD` be used. 
     "code": {
-      // this will be the precise code
-      "source": "..."
+      "source": "PROJECT_CODE",
+      // To define a hierarchical classification, the "parent" attribute refers to the attribute "above" it in the hierarchy.
+      // Set it on the `code` field, not a title field. See the ["Labels and Hierarchies" example][/examples/labels-and-hierarchies/].
+      "parent": "PROGRAMME_NAME"
```

Comment:
Is it clear enough that this must be the id / code for the programme not just its name ...?